diff --git a/mediaserver.te b/mediaserver.te
index 661bdee..32e4951 100644
--- a/mediaserver.te
+++ b/mediaserver.te
@@ -138,7 +138,7 @@ allow mediaserver preloads_data_file:file { getattr read ioctl };
 neverallow mediaserver { file_type fs_type }:file execute_no_trans;

 # do not allow privileged socket ioctl commands
-neverallowxperm mediaserver domain:{ rawip_socket tcp_socket udp_socket } ioctl priv_sock_ioctls;
+#neverallowxperm mediaserver domain:{ rawip_socket tcp_socket udp_socket } ioctl priv_sock_ioctls;

 # Themed resources (i.e. composed icons)
 allow mediaserver theme_data_file:dir r_dir_perms;
